api/v1beta1/bgpadvertisement_webhook.go

```diff
@@ -56,8 +56,13 @@ func (bgpAdv *BGPAdvertisement) ValidateCreate() error {
 		return err
 	}
 
+	communities, err := getExistingCommunities()
```

Comment:
Same for the commit message, try to be impersonal.
Also, nit in existing

Reply:
what is the nit with getExistingCommunities?